src/components/Rubric/hooks.test.ts

```diff
@@ -0,0 +1,117 @@
+import { mocked } from 'ts-jest/utils';
+
+import {
+  usePageData,
+  useRubricConfig,
+} from 'data/services/lms/hooks/selectors';
+import { mockUseKeyedState } from '@edx/react-unit-test-utils';
+import { submitRubric } from 'data/services/lms/hooks/actions';
+import { useRubricData, stateKeys } from './hooks';
+
+jest.mock('data/services/lms/hooks/selectors');
+jest.mock('data/services/lms/hooks/actions');
+
+const mockedUsePageData = mocked(usePageData);
+const mockedUseRubricConfig= mocked(useRubricConfig);
+const mockedSubmitRubric = mocked(submitRubric);
+
+const states = mockUseKeyedState(stateKeys);
```

Comment:
convention is to use `state`... is there a reason to have this file be different? otherwise would rather keep convention for consistency